zebedee/client.go

```diff
@@ -0,0 +1,302 @@
+package zebedee
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"path/filepath"
+	"strings"
+	"sync"
+	"time"
+
+	"github.com/ONSdigital/dp-frontend-models/model"
```

Comment:
this is a slightly misplaced comment, took a while to notice...

we should probably split the "zebedee client" and "preparing data for the frontend renderer" into two separate packages

there are probably a lot of use cases where we want to get data from zebedee (e.g. the router), but don't want it automatically mapped to the frontend models since it'll never get to the renderer

splitting into two packages (client, and mapping to renderer models) means callers only need to import dependencies which are absolutely necessary